### PR TITLE
Moved button

### LINKED ISSUE #118: Move logout button to top right
**Is your feature request related to a problem? Please describe.**
Client wants this.

**Describe the solution you'd like**
just change position.

### PR DESCRIPTION
**Pull request by sspangenberg**
Moved logout button to the top right corner

closes #118

### CODE REVIEW

**Comment by schmidtjonas** [comment]
Please edit your message and comment on what you did.

File: TelegramClient-UI.package/TCUMain.class/instance/addHeader.st [hunk 1]
```diff
@@ -13,7 +11,7 @@ addHeader
 	logoutButton := TCUButton new
 	 	text: 'logout';
 		on:#mouseUp send: #logout to: self;
-		position: header position + 5.
+		position: (header position x + header width - 105) @ 5.
```
Comment (schmidtjonas) [inline TelegramClient-UI.package/TCUMain.class/instance/addHeader.st:14]: try and remove that 105 by using a `layoutPolicy` as used in `TCUChatsList >> initializeLayout`

File: TelegramClient-UI.package/TCUMain.class/instance/addHeader.st [hunk 2]
```diff
@@ -13,7 +11,7 @@ addHeader
 	logoutButton := TCUButton new
 	 	text: 'logout';
 		on:#mouseUp send: #logout to: self;
```
Comment (schmidtjonas) [inline TelegramClient-UI.package/TCUMain.class/instance/addHeader.st:13]: please add space before messages

**Comment by rohansaw** [comment]
Please fix code standards here!

File: TelegramClient-UI.package/TCUMain.class/instance/addHeader.st (hunk 1 above)
Comment (sspangenberg) [reply TelegramClient-UI.package/TCUMain.class/instance/addHeader.st]: `layoutPolicy` not working well here because of the centered title

**Comment by schmidtjonas** [comment]
Please resolve merge conflicts

**Pull request merged**